chrome/content/zotero/itemTree.jsx

```diff
@@ -705,6 +705,16 @@ var ItemTree = class ItemTree extends LibraryTree {
 							sort = id;
 						}
 					}
+					// If a child item is modified and its parent's row is expanded,
+					// collapse and re-open the parent.
```

Comment:
Well, that's not really what this is, right? It's specifically a child item that was in the trash — and therefore not in the current view, which is why it doesn't match the `row !== undefined` block above — that was modified to no longer be in the trash, so we have to close and reopen the parent item to refresh the list of child items.

Reply:
Right, looking at it now, the comment does sound way broader than I intended. How about this?
```
// If a trashed child item is restored while its parent's row is expanded,
// collapse and re-open the parent to have that child item row added.
```